src/resolve/resolved-dns-transaction.h

```diff
@@ -1,8 +1,10 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
-#include "sd-event.h"
 #include "in-addr-util.h"
+#include "sd-event.h"
```

Comment:
sd-*.h are our public headers, we always include them first, hence please do not reorder this.

Reply:
Fixed, thanks.